backend/server/routes.js

```diff
@@ -17,16 +18,41 @@ module.exports = {
       res.send('{"message":"Developer Profiles API"}');
     });
 
+    server.use('/api/saveresethash', forgotPasswordRouter);
+
     // In production build all other requests are handled by the frontend
     if (process.env.NODE_ENV === 'production') {
       server.get('*', (req, res) => {
-        res.sendFile(path.resolve(__dirname, '../../frontend/lambda-in/build', 'index.html'));
+        res.sendFile(
+          path.resolve(
+            __dirname,
+            '../../frontend/lambda-in/build',
+            'index.html',
+          ),
+        );
       });
     } else {
       server.get('*', (req, res) => {
         res.set('Content-Type', 'application/json');
-        res.send('{"message":"On production build this action will result in a redirect to the frontend."}');
+        res.send(
+          '{"message":"On production build this action will result in a redirect to the frontend."}',
+        );
       });
     }
   },
+  private: (server) => {
+    // This serves the Seekers (Employees) DB. It allows GET, POST, PUT and DELETE
+    server.use('/api/seekers', seekersRouter);
+    // This serves the Employers DB. It allows GET, POST, PUT and DELETE
+    server.use('/api/employers', employerRouter);
+    // server.use('/api/resetPassword', forgotPasswordRouter);
+  },
+  payments: {
+    /**
+     * Define an endpoint to process Stripe payment requests
+     */
+    stripe: (server) => {
+      server.use('/payments/stripe', stripeRouter);
+    },
+  },
 };
```

Comment:
This file was not merged correctly

Reply:
It is mixing the old router.js file with the new one.

Reply:
I am not sure, what's wrong. What do I need to do?